nodejs/week2/meal-sharing/src/backend/api/review-router.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const router = express.Router();
+
+const reviews = require("./../data/reviews.json");
+
+//...............Respond with the json for reviews with the corresponding id
+
+router.get("/:id", async (request, response) => {
+  try {
+    // knex syntax for selecting things. Look up the documentation for knex for further info
+      const realId = parseInt(request.params.id) - 1;
+      response.send(reviews[realId]);
```

Comment:
Also here you shouldn't use the array index :)

Reply:
I get confused.:) maybe I did not underestand the question. Would you please let me know what is the problem with my method of solving.

Reply:
I got it now your meaning and I am in the process to solve the problem. Thank you Kasper